src/hAPI/README.md

```diff
@@ -1,5 +1,54 @@
-# hAPI TypeScript Library 
+# What is this?
 
-## Acknowledgements 
+This repository contains the necessary components to run the multimodal audio-haptic photo renderer in the IMAGE extension using the Haply 2DIY device.
 
-The [hAPI library](https://github.com/Shared-Reality-Lab/IMAGE-browser/tree/main/src/hAPI) used in the project is a TypesSript port of the [Java hAPI library](https://gitlab.com/Haply/hAPI) originally authored by Oliver Anthony, [Haply Robotics](https://haply.co/). 
+## Requirements
+
+* A Haply 2DIY with its firmware installed.
+## How To Use
+
+* Ensure the Docker container hosting the appropriate haptic handler is running on the development / main server.
+* Connect the Haply 2DIY device through a serial USB connection.
+* In the extension settings under the "Advanced Options", turn on Developer Mode and select the "Haply 2diy" option. Change the server URL if needed.
+* Get an IMAGE rendering of a photograph, wait for a response and then and check if the list of available renderers in the pop-up window contains a photo audio-haptic renderer. If not, recheck the first three steps, and/or try obtaining an IMAGE rendering for another photograph.
+* Click the "Connect To Haply" button and select the COM port for the Haply device. Steadily grasp the 2DIY's knob and click the "START" button to initiate the audio-haptic experience.
+* The audio-haptic experience is equivalent to that shown in state diagram below, i.e., it plays a TTS narration for each segment, followed by a sonified experience of all subsegments within that segment, and then a haptically guided tour of each subsegment with the 2DIY. Once all segments are covered, the same process is repeated for objects, starting with grouped objects and then ungrouped objects. Subsegments are defined as isolated contours of a segment, as shown in the image below.
+* The Previous, Next, and Stop buttons can be used to navigate back and forth in the interaction, which always follows the pattern:
+![a state machine diagram explaining how the audio-haptic experience works](./images/haptics_state_machine.png){:height="250px"}
+NOTE:  this diagram will be changed slighly to indicate a starting state
```

Comment:
Bringing this note back to your attention.

Reply:
Removed, and whoops, that was a draft image and not the one intended to be shown.